apps/viewer/src/components/TypebotPageV3.tsx

```diff
@@ -54,11 +58,89 @@ export const TypebotPageV3 = ({
       }}
     >
       <SEO url={url} typebotName={name} metadata={metadata} />
-      <Standard
-        typebot={publicId}
-        onInit={clearQueryParamsIfNecessary}
-        font={font ?? undefined}
-      />
+
+      {isPasswordProtected || !isPasswordValid ? (
+        <div
```

Comment:
Since Chakra UI is not available here, ended up doing regular html css to make this form